node/basicnode/list.go

```diff
@@ -31,6 +31,9 @@ func (plainList) LookupByNode(datamodel.Node) (datamodel.Node, error) {
 	return mixins.List{TypeName: "list"}.LookupByNode(nil)
 }
 func (n *plainList) LookupByIndex(idx int64) (datamodel.Node, error) {
+	if n.Length() < 0 {
```

Comment:
do you want if the length is less than 0 here, or if `idx < 0` ?

Reply:
oof of course, fixed it